android/app/src/main/java/org/fossasia/openevent/utils/CompressImage.java

```diff
@@ -0,0 +1,30 @@
+package org.fossasia.openevent.utils;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Created by free4murad on 3/9/17.
+ */
+
+public class CompressImage {
+    public static String compressImage(String file) {
+        URL url;
+        String temp = "";
+        int indexOfPath = -1;
+        try {
+            url = new URL(file);
+        } catch (MalformedURLException e) {
+            return file;
+        }
+        indexOfPath = file.indexOf(url.getPath());
+        if(!url.getHost().endsWith(".rsz.io")) {
+            temp = file.substring(0, indexOfPath) + ".rsz.io" + file.substring(indexOfPath) + "?width=300";
+            file = temp;
+        }
+        if(url.getProtocol().equals("https")){
+            file = temp.replace("https", "http");
```

Comment:
@the-dagger @mananwason @harshitagupta30 
To use string resources in `CompressImage.java` we need to pass context of the activity as a parameter.
It will require a editing functions at many levels. 
1. Activity classes 2. Adapter classes 3. Speaker class.
We can totally do this but I think its not necessary.
These strings need not be translated.

What do you guys think?